Odata-docs/client/v6/using_blazor_wasm_with_odata_client.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Blazor Wasm with OData Client"
+description: ""
+
+author: elokerio
+ms.author: elokerio
+ms.date: 1/18/2021
+ms.topic: article
+ 
+---
+# How To Use Blazor Web Assembly with OData Client 
+**Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v6.md)]
```

Comment:
A few things to note:
`author` - should be your github username.

I update the docs to v7 
`**Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v7.md)]`

The file should be in `/Odata-docs/client/using_blazor_wasm_with_odata_client.md`

Reply:
resolved